default/stringtables/en.txt

```diff
@@ -1402,6 +1402,15 @@ Count of minutes after the cookie record will be considered expired.
 OPTIONS_DB_PUBLISH_STATISTICS
 Enable sending empire staticstics to the player.
 
+OPTIONS_DB_FIRST_TURN_TIME
+Absolute time point in seconds since epoch. If 0 first turn advance will happen after expiring interval.
+
+OPTIONS_DB_TIMEOUT_INTERVAL
+Maximum interval in seconds between turn advances. If 0 turn advances by timeout are disabled.
+
+OPTIONS_DB_TIMEOUT_FIXED_INTERVAL
```

Comment:
Not sure about the "If enabled"... What's the standard for other optipns that toggle something?

Regardless, maybe "Turns advance after fixed intervals, regardless of when and whether players have submitted turn orders."

Reply:
Yep, there no "If enabled" in other toggle options.